v4/export/sql.go

```diff
@@ -275,6 +282,41 @@ func SetTiDBSnapshot(db *sql.DB, snapshot string) error {
 	return withStack(err)
 }
 
+func existsGeneratedFields(db *sql.DB, dbName, tableName string) (bool, error) {
```

Comment:
This function seems unnecessary. `buildSelectField` can call `buildInsertableFields` directly. The side effect skipping this check is we'll always run `SELECT all, the, columns` rather than `SELECT *`, but I don't think that's a bad thing.

Reply:
I also think we don't need it

Reply:
sure, put logic into [`buildSelectField`](https://github.com/pingcap/dumpling/pull/47/files#diff-5c2ce07cc844075b8e4dfe29c8e256f1R286), use one query to achieve that